compiler/ast/src/common/identifier.rs

```diff
@@ -87,8 +87,16 @@ impl<N: Network> From<&IdentifierCore<N>> for Identifier {
     }
 }
 
+// Converts an `Identifier` to a `Path` expression
 impl From<Identifier> for Expression {
     fn from(value: Identifier) -> Self {
-        Expression::Identifier(value)
+        Expression::Path(crate::Path::from(value))
+    }
+}
+
+// Converts an `Identifier` to a `Path`
+impl From<Identifier> for Path {
+    fn from(value: Identifier) -> Self {
+        Path { segments: vec![value], absolute_path: Some(vec![value.name]), span: value.span, id: value.id }
```

Comment:
Why is the identifier considered an absolute path?
Mainly for my own understanding, but could be helpful to leave a comment.